certificate-policy.md

```diff
@@ -841,289 +811,247 @@ All persons filling trusted roles shall be selected on the basis of loyalty, tru
 
 ### 5.4.5 Audit Log Backup Procedures
-Audit logs and audit summaries shall be backed up at least monthly.  A copy of the audit log shall be sent off-site on a monthly basis.
+Audit logs and audit summaries shall be backed up at least monthly.  Copies of the audit logs shall be sent off-site on a monthly basis.
 
 ### 5.4.6 Audit Log Accumulation System (internal vs. external)
-The audit log collection system may or may not be external to the CA system.  Automated audit processes shall be invoked at system or application startup, and cease only at system or application shutdown.  Audit collection systems shall be configured such that security audit data is protected against loss (e.g., overwriting or overflow of automated log files).  Should it become apparent that an automated audit system has failed, and the integrity of the system or confidentiality of the information protected by the system is at risk, operations shall be suspended until the problem has been remedied.
+The audit log collection system may or may not be external to the CA system.  Automated audit processes shall be invoked at system or application startup, and cease only at system or application shutdown.  Audit collection systems shall be configured such that security audit data is protected against loss (e.g. overwriting or overflow of automated log files).  Should it become apparent that an automated audit system has failed, and the integrity of the system or confidentiality of the information protected by the system is at risk, operations shall be suspended until the problem has been remedied.
 
 ### 5.4.7 Notification to event-causing subject
 There is no requirement to notify a subject that an event was audited.  Real-time alerts are neither required nor prohibited by this policy.
 
 ### 5.4.8 Vulnerability assessments
-Additionally, the CA's security program shall include an annual Risk Assessment that:
+The CA's security program shall include an annual Risk Assessment that:
+
+1. Identifies foreseeable internal and external threats that could result in unauthorized access, disclosure, misuse, alteration, or destruction of any Certificate data or Certificate management processes
+2. Assesses the likelihood and potential damage of these threats, taking into consideration the sensitivity of the Certificate data and Certificate management processes 
+3. Assesses the sufficiency of the policies, procedures, information systems, technology, and other arrangements that the CA has in place to counter such threats
+
+The CA shall undergo or perform a Vulnerability Scan:
+- At least once per quarter, on public and private IP addresses identified as within the CA's system boundaries
+- Within one week of receiving a request from the FPKIPA or the U.S. Government Federal Information Security Modernization Act Authorizing Official for the CA system or the CA/B Forum
```

Comment:
I don't think the CA/B Forum is the right entity here. They don't play an active enforcement or oversight role on individual CAs, only root programs do. So I think we want to have this reference root program representatives, rather than the CA/B Forum.

Reply:
I too questioned this one.  The original verbiage referencing CA/B Forum is in the Baseline Requirements; I updated for USG and authorities (FISMA and PA by delegation from CIO council).  I did not see any trust store program (from microsoft, mozilla, java, etc) policy that overrode the original Baseline Requirements.    

I'm marking this for review by general counsel.  We should also review the CA/B Forum bylaws to determine how a CA could be directed (e.g. through a Vote?) to perform penetration testing, and the legal jurisdiction. 

[I'm removing the CA/B Forum for now.]